--- src/serviceframework/ipc/qmetaobjectbuilder_p.h.fix_qmetaobjectbuilder	2011-05-10 09:06:12.000000000 +0100
+++ src/serviceframework/ipc/qmetaobjectbuilder_p.h	2011-10-16 18:24:26.373281438 +0100
@@ -172,7 +172,7 @@ public:
     int indexOfEnumerator(const QByteArray& name);
     int indexOfClassInfo(const QByteArray& name);
 
-    typedef int (*StaticMetacallFunction)(QMetaObject::Call, int, void **);
+    typedef QMetaObjectExtraData::StaticMetacallFunction StaticMetacallFunction;
 
     QMetaObjectBuilder::StaticMetacallFunction staticMetacallFunction() const;
     void setStaticMetacallFunction(QMetaObjectBuilder::StaticMetacallFunction value);
