diff -urN qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qaudioinput_pulse.cpp qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qaudioinput_pulse.cpp
--- qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qaudioinput_pulse.cpp	2011-05-10 19:06:09.000000000 +1100
+++ qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qaudioinput_pulse.cpp	2013-03-17 00:04:03.371106853 +1100
@@ -47,6 +47,8 @@
 #include "qpulseaudioengine.h"
 #include "qpulsehelpers.h"
 
+#include <unistd.h>
+
 QT_BEGIN_NAMESPACE
 
 const int PeriodTimeMs = 50;
diff -urN qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qaudiooutput_pulse.cpp qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qaudiooutput_pulse.cpp
--- qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qaudiooutput_pulse.cpp	2011-05-10 19:06:09.000000000 +1100
+++ qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qaudiooutput_pulse.cpp	2013-03-17 00:04:03.371606854 +1100
@@ -47,6 +47,8 @@
 #include "qpulseaudioengine.h"
 #include "qpulsehelpers.h"
 
+#include <unistd.h>
+
 QT_BEGIN_NAMESPACE
 
 const int PeriodTimeMs = 20;
diff -urN qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qpulseaudioengine.cpp qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qpulseaudioengine.cpp
--- qt-mobility-opensource-src-1.2.0/plugins/multimedia/pulseaudio/qpulseaudioengine.cpp	2011-05-10 19:06:09.000000000 +1100
+++ qt-mobility-opensource-src-1.2.0-patched/plugins/multimedia/pulseaudio/qpulseaudioengine.cpp	2013-03-17 00:04:03.371606854 +1100
@@ -47,6 +47,8 @@
 #include "qaudiooutput_pulse.h"
 #include "qpulsehelpers.h"
 
+#include <unistd.h>
+
 QT_BEGIN_NAMESPACE
 
 static void serverInfoCallback(pa_context *context, const pa_server_info *info, void *userdata)
